concepticondata/conceptlists/BeijingDaxue-1964-905.tsv

```diff
@@ -254,7 +254,7 @@ BeijingDaxue-1964-905-252	252	窗子	126	noun	living, dwelling	八、房屋	名
 BeijingDaxue-1964-905-253	253	鬥檻兒	127	noun	living, dwelling	八、房屋	名 詞	bucket trap	dòu kǎn ér			斗槛儿
 BeijingDaxue-1964-905-254	254	曬臺	127	noun	living, dwelling	八、房屋	名 詞	flat roof	shài tái			晒台
 BeijingDaxue-1964-905-255	255	欄桿	128	noun	living, dwelling	八、房屋	名 詞	balustrade	lán gǎn			栏杆
-BeijingDaxue-1964-905-256	256	臺階	128	noun	living, dwelling	八、房屋	名 詞	steps (of stairs)	tái jiē	1158	STEP	台阶
+BeijingDaxue-1964-905-256	256	臺階	128	noun	living, dwelling	八、房屋	名 詞	steps (of stairs)	tái jiē			台阶
```

Comment:
why would you not allow that link? 

```

  | 0
-- | --
step | 1158 | STEP | A flat surface or support that facilitates the ascent or descent of a stairway.
```
The Chinese word means [Stufe](http://www.handedict.de/chinesisch_deutsch.php?e=19442). So I would ask to not delete this.

Reply:
Mei-Shin told me it only refers to outside stairs, e.g. a Türschwelle or a  stairway leading up to a building or up a hill.

Reply:
Really? Google says [differently](https://www.google.com/search?q=%E6%A5%BC%E6%A2%AF%E5%8F%B0%E9%98%B6). I think it is good enough, since there are no other words for Stufe, even if there are nuances, but the single-level translation is not the main reference point, but the intended use in the questionnaire. And they obviously asked for "steps“ there.

Reply:
It's also what it seems to say on Wikipedia: https://zh.wikipedia.org/wiki/%E5%8F%B0%E9%98%B6
But I agree that it most likely was intended to mean Stufe. I could adjust the definition to include either a single step or a set of steps (our STAIRWAY is defined to refer to stairs leading from one floor to another one)

Reply:
Just re-insert it for the time being, and it's fine. Thanks!

Reply:
Re-added.